### ISSUE TITLE
Snapcast server settings are not stored

### ISSUE THREAD

**Issue by Links2004** [cristianpb/muse#15]
Is there a way to configure the Snapcast server settings in a config in the backend?
every time I reload the page the settings are back to default which does not match my setup.

Note:
I have a nginx before my snapcast to get ssl / https support and my mopidy and snapcast are running both on http port 443 via hostname.

**Comment by Links2004** [comment]
looks like "Mopidy server" has the same limitations, and on top setting https does not change the websocket URL from ws://... to wss:// which than is blocked by the browser 
```
Mixed Content: The page at '<URL>' was loaded over HTTPS, but attempted to connect to the insecure WebSocket endpoint 'ws:<URL>/mopidy/ws/'. This request has been blocked; this endpoint must be available over WSS.
```

**Comment by cristianpb** [comment]
Hi, 
Thanks for noticing the bug about "mopidy server" ssl option. It should be solved in 499722b and included in next release. 

It should be possible to save server settings in the backend, I just have to know how to implement it.

**Comment by Links2004** [comment]
may this helps to get the idea for using the config file:
https://github.com/jaedb/Iris/search?q=spotify_authorization_url&unscoped_q=spotify_authorization_url

**Comment by cristianpb** [comment]
There is now the possibility to configure snapcast and mopidy server settings in the backend config. You can add the following in your `mopidy.conf` file

```config
[muse]
enabled = true
# the following are optional values
mopidy_host = localhost
mopidy_port = 6680
mopidy_ssl = false
snapcast_host = localhost
snapcast_port = 1780
snapcast_ssl = false
```

This is available from version v0.0.19 of mopidy-muse.

**Comment by Links2004** [comment]
thanks for picking up on the config storage,

I updated but it looks like snapcast has some problem, looks like true is not mapped to 's'.

```
[Snapcast]: catch error: DOMException: Failed to construct 'WebSocket': The URL's scheme must be either 'ws' or 'wss'. 'wstrue' is not allowed.
```

```ini
[muse]
enabled = true
mopidy_host = mopidy.audio
mopidy_port = 443
mopidy_ssl = true
snapcast_host = snapcast.audio
snapcast_port = 443
snapcast_ssl = true
```

**Comment by cristianpb** [comment]
Yes, I saw that too after making the release 0.0.19. 

I just released version 0.0.20, it should work well now.

Please let me know, if it's not the case

**Comment by Links2004** [comment]
yes working fine now.
